src/annotator/integrations/test/pdf-metadata-test.js

```diff
@@ -1,6 +1,7 @@
 import { delay } from '@hypothesis/frontend-testing';
 import EventEmitter from 'tiny-emitter';
 
+import { promiseWithResolvers } from '../../../shared/promise-with-resolvers';
```

Comment:
We could replace this with [`Promise.withResolvers`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/withResolvers) going forwards.